newrelic/resource_newrelic_nrql_alert_condition.go

```diff
@@ -274,7 +273,9 @@ func resourceNewRelicNrqlAlertCondition() *schema.Resource {
 			},
 			"violation_time_limit": {
 				Type:          schema.TypeString,
+				Deprecated:    "use `violation_time_limit_seconds` attribute instead",
 				Optional:      true,
+				Computed:      true,
```

Comment:
I added a computed to this field to prevent drift when using `violation_time_limit_seconds`.